src/core/profile_manager.mjs

```diff
@@ -148,15 +149,12 @@ export class ProfileManager {
   resourcesForConsensusPod (profile, nodeIds, yamlRoot) {
     if (!profile) throw new MissingArgumentError('profile is required')
 
-    // prepare name and account IDs for nodes
-    const realm = constants.HEDERA_NODE_ACCOUNT_ID_START.realm
-    const shard = constants.HEDERA_NODE_ACCOUNT_ID_START.shard
-    let accountId = constants.HEDERA_NODE_ACCOUNT_ID_START.num
+    const accountMap = getNodeAccountMap(nodeIds)
```

Comment:
there are some changes I'm introducing that will cause us to run this logic before the helm charts have been installed, so this command will fail.   Please revert for the change in this location.

Reply:
I need your changes, so I'll just revert this in mine when I need it.